bundler.d/test.rb

```diff
@@ -13,4 +13,5 @@
   gem 'factory_girl_rails', '~> 4.5', :require => false
   gem 'rubocop-checkstyle_formatter', '~> 0.1'
   gem "poltergeist"
+  gem 'shoulda-matchers'
```

Comment:
This should be pinned to a major version, as v3 will require an additional configuration step.